script.js

```diff
@@ -0,0 +1,94 @@
+// navigation Script
+const menuIcon = document.querySelector('.fa-bars');
+const mobileMenuList = document.querySelector('.mobile-menu-pane');
+const mobileHeader = document.querySelector('.intro');
+const bodyS = document.querySelector('.body');
+
+menuIcon.addEventListener('click', () => {
+  mobileHeader.classList.toggle('header-bg');
+  menuIcon.classList.toggle('fa-xmark');
+  mobileMenuList.classList.toggle('opened');
+  bodyS.classList.toggle('stopScroll');
+});
+
+function removeStuffs() {
+  mobileMenuList.classList.remove('opened');
+  menuIcon.classList.remove('fa-xmark');
+  mobileHeader.classList.remove('header-bg');
+  bodyS.classList.remove('stopScroll');
+}
+
+// Speaker object prototype
+function Speaker(fullname, title, desc, img) {
+  this.fullname = fullname;
+  this.title = title;
+  this.desc = desc;
+  this.img = img;
+}
+
+function getSpeakers() {
+// create array to store projects
+  let speakers = [];
+  const speakerInfo1 = new Speaker('Dr Mahamudu Bawumia', ' Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/alhaji-mahamudu-bawumia.jpeg');
+  const speakerInfo2 = new Speaker('Hon. Ursula Owusu', 'Minister of Communication and Digital Economy', 'A Design Strategist and a Tech Enthusiast', 'img/ursula-owusu.hpeg.jpg');
+  const speakerInfo3 = new Speaker('Hon. Dr. Kwaku Afriyie', 'Minister, Environment, Science, Technology and Innovation (MESTI)', 'A Design Strategist and a Tech Enthusiast', 'img/hon-kwaku-afriyie.jpeg');
+  const speakerInfo4 = new Speaker('Dr Gordon Adomdza', 'Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/gordon.jpg');
+  speakers = [speakerInfo1, speakerInfo2, speakerInfo3, speakerInfo4];
```

Comment:
Kindly make sure that your project sticks as much as possible to the template provided. Your project should render dynamically six elements and your project is rendering just 4 elements.  This is a requirement of the project.  :scroll: :scroll: